src/octets.rs

```diff
@@ -66,11 +66,12 @@ macro_rules! peek_u {
         }
 
         let mut out: $ty = 0;
+        let len = $len;
```

Comment:
It'd probably be nicer to put this above the `if src.len() < $len` (i.e. immediately before or after `let src = ...`) for consistency, so we would use `len` instead of `$len` everywhere.

Reply:
yeah I missed that, thanks.

Reply:
Updated.